api/src/shared/domain/models/CertificationCandidate.js

```diff
@@ -104,6 +105,10 @@ class CertificationCandidate {
   isEnrolledToComplementaryOnly() {
     return this.subscriptions.length === 1 && this.subscriptions[0].isComplementary();
   }
+
+  isEnrolledToDoubleCertification() {
+    return this.complementaryCertification?.key === ComplementaryCertificationKeys.CLEA;
```

Comment:
Histoire d'être explicite sur ce qu'est une double certif, au delà que ce soit CLEA

```suggestion
    return this.subscriptions.length === 2 && this.complementaryCertification?.key === ComplementaryCertificationKeys.CLEA;
```